Frontend/ui-common/src/Components/PopUpModal/Controller.purs

```diff
@@ -34,6 +36,7 @@ data Action = OnButton1Click
             | OnImageClick
             | Tipbtnclick Int Int
             | DismissPopup
+            | OnClose PopUpAction  ------- OnClose (1 if triggered by btn1 else 2)
```

Comment:
`OnClose (1 if triggered by btn1 else 2)` is it still 0 1 2 3?

Reply:
No , this comment has to be removed.